src/plugins/intel_gpu/src/graph/include/kernel_selector_helper.h

```diff
@@ -215,6 +216,9 @@ inline params_t get_default_params(const kernel_impl_params& param_info) {
 
     params.inputs[0] = convert_data_tensor(input_layout);
     params.outputs[0] = convert_data_tensor(output_layout);
+    if (use_shape_agnostic(param_info)) {
```

Comment:
It is really minor.. but the kernel "is" shape agnostic, not "use".. could you please rename?

Reply:
done